notebooks/20230105_overview_plot.ipynb

```diff
@@ -4,11 +4,13 @@
    "cell_type": "markdown",
```

Comment:
<p>ah very nice, didn't realize you had made this with code i thought it was hand made</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/2022-sourmashconsumr-validation/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>